### ISSUE TITLE
Deprecate in-memory recordsScanned counter for "Built Range" logs

### ISSUE THREAD

**Issue by nblintao** [FoundationDB/fdb-record-layer#1125]
Today, `OnlineIndexer` uses an in-memory counter `recordsScanned` to log how many records have been scanned ("Built Range" logs) once every 5 minutes, while it uses a value persisted in `scannedRecordsSubspace` for `IndexBuildState` requests.

We should deprecate `recordsScanned` and just use `scannedRecordsSubspace` in "Built Range" logs. And we can also add `recordsInTotal` to that log. In fact, all fields of `IndexBuildState` could be there.

**Comment by vasantteja** [comment]
Can you please assign this to me?

**Comment by nblintao** [comment]
@vasantteja Of course. Thanks for being interested! Please let me you if you have any question.

**Comment by nblintao** [comment]
Hi @vasantteja, any progress? Do you need any help?

**Comment by vasantteja** [comment]
Should I create another branch or should I fork this and make the changes on those?

**Comment by vasantteja** [comment]
@nblintao can you please let me know how you want me to proceed?

**Comment by nblintao** [comment]
@vasantteja Sorry I missed your earlier message. You should:

1. fork this repository as your own repository
2. create a new branch on your own repository to develop
3. After you finish, create a Pull Request to the main branch of this repository from the feature branch of your own repository.

And this will be helpful as well. https://github.com/FoundationDB/fdb-record-layer/blob/main/CONTRIBUTING.md#opening-a-pull-request

**Comment by nblintao** [comment]
@vasantteja For the general workflow of making a pull request, I find this might be helpful https://github.com/firstcontributions/first-contributions

**Comment by nblintao** [comment]
Hi @vasantteja, do you still intend to work on this issue?

**Comment by vasantteja** [comment]
@nblintao  hey yeah. I intend to work on this.
